react/features/conference/components/Conference.native.js

```diff
@@ -232,12 +259,41 @@ class Conference extends Component {
 }
 
 /**
+ * Maps dispatching of some action to React component props.
+ *
+ * @param {Function} dispatch - Redux action dispatcher.
+ * @private
+ * @returns {{
+ *     _onConnect: Function,
+ *     _onDisconnect: Function,
+ *     _onToolbarVisibilityChange: Function
+ * }}
+ */
+function _mapDispatchToProps(dispatch) {
+    return {
+        _onConnect() {
```

Comment:
Ok, let's discuss it on more general terms. As a general rule, I usually use onXXX for functions that are passed around and called by some other entity when some event happens, not by directly calling them myself.

In this case, onConnect suggests me that the connection was made, but it actually wasn't, the component was mounted and we want to connect after that.